task_sdk/src/airflow/sdk/execution_time/supervisor.py

```diff
@@ -342,10 +345,11 @@ def start(
 
             # Python GC should delete these for us, but lets make double sure that we don't keep anything
             # around in the forked processes, especially things that might involve open files or sockets!
-            del path
             del client
-            del what
             del logger
+            for k in list(on_child_started_kwargs.keys()):
+                v = on_child_started_kwargs.pop(k)
+                del v
```

Comment:
I think you can just do this
```suggestion
            del on_child_started_kwargs
```

Reply:
haha ok. certainly easier